From cc69c9c85b7e1ff61c0c4cd5b21b299ba6e00b8c Mon Sep 17 00:00:00 2001
From: Calum Lind <calumlind+deluge@gmail.com>
Date: Mon, 20 Mar 2017 08:29:35 +0000
Subject: [PATCH] [#2990] [Core] Fix torrent priorities mismatch

 * The old priorities instead of updated call to lt were being saved to self.options.
---
 deluge/core/torrent.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deluge/core/torrent.py b/deluge/core/torrent.py
index f5c6c79..3b1a1e0 100644
--- a/deluge/core/torrent.py
+++ b/deluge/core/torrent.py
@@ -334,7 +334,7 @@ def set_file_priorities(self, file_priorities):
                     self.update_state()
                     break
 
-        self.options["file_priorities"] = handle_file_priorities
+        self.options["file_priorities"] = self.handle.file_priorities()
         if self.options["file_priorities"] != list(file_priorities):
             log.warning("File priorities were not set for this torrent")
 
